scripts/us_eia/opendata/process/copy.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Directory names
+ELEC=electricity
+NG=natural_gas
+PET=petroleum
+INTL=international
+SEDS=seds
+TOTAL=total_energy
+
+if [[ "$#" -ne 1 ]]; then
+  echo "Usage: $0 <dataset>"
+  exit 1
+fi
+
+function copy() {
+  dataset=$1
+  src=tmp_raw_data/${dataset}/${dataset}
+  dst=${!dataset}
+
+  if [[ -z "$dst" ]]; then
+    echo "ERROR: Incorrect dataset"
+    exit 1
+  fi
+
+  set -x
+  fileutil cp -f --parallel_copy 3 ${src}.mcf ${src}.tmcf ${src}.csv /cns/jv-d/home/datcom/v3_mcf/us_eia/${dst}/
```

Comment:
I'm not sure if this script belongs in the data repo?

Reply:
Removed it.  Need to find some place for these g3-specific glue scripts :(